os/mm/umm_heap/umm_realloc.c

```diff
@@ -157,6 +167,6 @@ FAR void *realloc(FAR void *oldmem, size_t size)
 		}
```

Comment:
How about using new definitions for existed code, line 158?
```
for (heap_idx = HEAP_START_IDX; heap_idx <= HEAP_END_IDX; heap_idx++) {
```
There are several locations like above.

Reply:
Changed. Thanks.